bcda/cclf/cclf.go

```diff
@@ -228,15 +230,15 @@ func importCCLF(ctx context.Context, fileMetadata *cclfFileMetadata, importer im
 
 	if rawFile == nil {
 		fmt.Printf("File %s not found in archive %s.\n", fileMetadata.name, fileMetadata.filePath)
-		err = errors.Wrapf(err, "file %s not found in archive %s", fileMetadata.name, fileMetadata.filePath)
+		err = fmt.Errorf("file %s not found in archive %s", fileMetadata.name, fileMetadata.filePath)
```

Comment:
No longer call `errors.Wrap(err,...)` because the `err == nil` at this point. Calling `errors.Wrap(nil,...)`, returns a nil error.

See: https://github.com/pkg/errors/blob/master/errors.go#L185